.travis/install_test.sh

```diff
@@ -17,4 +17,4 @@ if [ "${COMPOSER_FLAGS}" = '--prefer-lowest' ]; then
     composer update --prefer-dist --no-interaction --prefer-stable --quiet
 fi
 
-composer update --prefer-dist --no-interaction --prefer-stable ${COMPOSER_FLAGS}
+travis_wait 20 composer update --prefer-dist --no-interaction --prefer-stable ${COMPOSER_FLAGS}
```

Comment:
dev-kit will make this disappear

Reply:
i just want to know how much time we need, i will remove it

Reply:
no luck =(
i do not want to debug `Composer\DependencyResolver\Solver::runSat` =)